forge-game/src/main/java/forge/game/zone/MagicStack.java

```diff
@@ -353,6 +353,40 @@ public final void add(SpellAbility sp, SpellAbilityStackInstance si, int id) {
             runParams.put(AbilityKey.CardLKI, lki);
             thisTurnCast.add(lki);
             sp.getActivatingPlayer().addSpellCastThisTurn();
+
+            // Add expend mana
+            Map<Player, Integer> players = Maps.newHashMap();
+
+            for(SpellAbility sa : sp.getPayingManaAbilities()) {
```

Comment:
Hmn, we might have to work with` payingMana` instead
otherwise if _Food Chain_ gets used twice (on different CMC) it'd miscalculate